README.md

```diff
@@ -1,37 +1,155 @@
 
-Tlf is a console (ncurses) mode general purpose CW/VOICE keyer, logging and 
-contest program for hamradio. 
+TLF is a logging program for radiosport, or ham radio contests. It supports CQWW, CQ-WPX, ARRL-DX, ARRL-FD, PACC, and EU SPRINT contests as well as many more, and also has general QSO and DXpedition modes. Users can add new contests by editing a text file.
 
-It supports the CQWW, the WPX, the ARRL-DX , the ARRL-FD, the PACC and the 
-EU SPRINT contests (single operator) as well as a LOT MORE basic contests, 
-general QSO and DXpedition mode. 
+TLF interfaces with a morse code generator, your sound card, a huge number of radios, and with a DX Cluster. It can project cluster data into the excellent Xplanet program, written by Hari Nair.
 
-It interfaces with a morse code generator, your sound card, a number of radios,
-and with a DX Cluster. 
+Contest operation mimics the popular TR-Log program for DOS, and the output file  is TR- as well as CABRILLO compatible. The user interface was designed with over 30 years of experience in CW contesting, and a group of active contesters continues to update the code. 
 
-Tlf can project cluster data into the excellent Xplanet program, written by 
-Hari Nair.
+TLF runs in console mode (a terminal window), allowing it to run on smaller machines, or remotely with limited bandwidth. Don't let the "retro" look of the console fool you, though; this is a fully modern contest logger that can support everything from a part time single op effort to a large multi-multi "big gun" operation.
 
-Contest operation mimics the popular TR-Log program for DOS, the output file 
-is TR- as well as CABRILLO compatible. The user interface was designed with 
-over 30 years of experience in CW contesting. 
+See Manual.md for an explanation of the user interface and detailed descriptions of setups for CW, RTTY, and SSB contesting. For specific questions once you've started using TLF, simply type `man tlf` at the command line. 
 
-The program was written for console mode on purpose, to make it run also on 
-smaller machines, or remotely via a modem link. 
+## Basic principles
 
-See `man tlf` for detailed documentation. See also doc/README for details, 
-and the INSTALL file for an quick installation guide.
+As a console program, there are no menus or pop-up windows in TLF. Instead, it uses two configuration files: `logcfg.dat` and a contest-specific rules file. Starting from an empty directory, it will read `<PREFIX>/logcfg.dat`, where `<PREFIX>` is the directory where the data files were installed (`/usr/local/share/tlf` by default). Any `logcfg.dat` or rules file stored in the local directory where TLF launches will override the default settings, which you probably want to do. See the **Quick Start** guide below for an example setup. The default `logcfg.dat` file has extensive comments documenting the settings, and can be opened in any text editor. At a minimum, you'll need to set your callsign.
 
+## Installation
+
```

Comment:
I'd add an "Installing from your distribution's repo" first as it is available in most distros. A first time user may not want to compile it from scratch.

Reply:
Now you tell me. I've just added this.